ra_aid/__main__.py

```
@@ -6,6 +6,7 @@
 from rich.console import Console
 from langgraph.checkpoint.memory import MemorySaver
 from ra_aid.env import validate_environment
+from ra_aid.project_info import get_project_info, format_project_info, display_project_status
```

Ruff (0.8.2)

`ra_aid.project_info.display_project_status` imported but unused

Remove unused import: `ra_aid.project_info.display_project_status`

(F401)